templates/firefox-ios.xliff

```diff
@@ -2650,14 +2641,14 @@
         <source>Thanks for choosing Firefox!</source>
         <note>Title for the first panel 'Welcome' in the First Run tour.</note>
       </trans-unit>
-      <trans-unit id="Sign in to Firefox">
-        <source>Sign in to Firefox</source>
-        <note>See http://mzl.la/1T8gxwo</note>
-      </trans-unit>
       <trans-unit id="Start Browsing">
         <source>Start Browsing</source>
         <note>See http://mzl.la/1T8gxwo</note>
       </trans-unit>
+      <trans-unit id="Turn on Sync...">
+        <source>Turn on Sync...</source>
+        <note>See http://mzl.la/1T8gxwo</note>
```

Comment:
Please add a small localization note here, adding to the link if you think it's still needed

Reply:
Also, please replace the three periods by a horizontal ellipsis character